source/PlayerInfoPanel.cpp

```diff
@@ -493,6 +493,7 @@ void PlayerInfoPanel::DrawPlayer(const Rectangle &bounds)
 		bright, Truncate::MIDDLE, true);
 	table.DrawTruncatedPair("net worth:", dim, Format::Credits(player.Accounts().NetWorth()) + " credits",
 		bright, Truncate::MIDDLE, true);
+	table.DrawTruncatedPair("time played:", dim, Format::PlayTime(player.GetPlayTime()), bright, Truncate::MIDDLE, true);
```

Comment:
Just following the format of the previous two table entries.
```suggestion
	table.DrawTruncatedPair("time played:", dim, Format::PlayTime(player.GetPlayTime()),
		bright, Truncate::MIDDLE, true);
```